Rakefile

```diff
@@ -15,7 +15,6 @@ RuboCop::RakeTask.new
 task default: %i[
   spec
   rubocop
-  generate_cops_documentation
```

Comment:
Why is this?

Reply:
I working on refactoring that task. I just don't want to run this task as default for now.
I'll re-add this after sometime (when the refactoring is done!).

Reason of needing to refactor is because the API changed, so that task has some false-positives now.

Reply:
Sure! In the future, I suggest to try to make this kind of changes as separate commits, and use the commit messages to properly explain the why. That helps future readers of the code understand where you're going :)

Reply:
Yeah, apologies on my part. It indeed wasn't friendly for the reviewer :sweat_smile: